win10_innosetup.iss

```diff
@@ -1,7 +1,7 @@
 ﻿
 
 #define MyAppName "pytripgui"
-#define MyAppVersion "X.Y.Z"
+#define MyAppVersion "1.2.0+rev28"
```

Comment:
Why this is changed ?
This file is a template, being filled by main.spec

![image](https://user-images.githubusercontent.com/7374733/128905409-2f59d78d-7d7a-4a22-81f5-310f689028a4.png)